evalai/utils/challenges.py

```diff
@@ -667,3 +669,62 @@ def display_leaderboard(challenge_id, phase_split_id):
         pretty_print_leaderboard_data(attributes, results)
     else:
         echo(style("Sorry, no Leaderboard results found.", bold=True, fg="red"))
+
+
+def upload_annotations_file_with_presigned_url(challenge_phase_pk, file_name):
```

Comment:
change this to `upload_presigned_url_annontations` or `upload_presigned_url_challenge_phase_annontations`?